apps/accelerate/open_alpha_tensor/open_alpha_tensor/core/actors/stage.py

```diff
@@ -368,7 +406,7 @@ def actor_prediction(
         torch.tensor([-1] * (len(policies) - 1) + [reward]), dim=0
     )
     if return_actions:
-        actions = [state_dict[hash_state][5] for hash_state in hash_states]
+        actions = [state_dict[hash_state][4] for hash_state in hash_states]
```

Comment:
please keep the old code modification (the code in red should replace the code in green)